rpc/types_broadcast_transaction.go

```diff
@@ -40,14 +42,26 @@ type BroadcastInvokev0Txn struct {
 	InvokeTxnV0
 }
 
+func (tx BroadcastInvokev0Txn) InvokeType() string {
```

Comment:
Can you also do this for the broadcastDeclare and broadcastDeployAccount types please?

Reply:
Sure I will do that
Should I create a separate PR for them?? To keep things segregated?

Reply:
Might as well do them all at once

Reply:
Mmm
I will create a separate one to keep things clean from my side :)